Changes

```diff
@@ -1,5 +1,31 @@
 Release history for Zonemaster component Zonemaster-Backend
 
+
+v5.0.0 2019-11-15
+
+ [API change]
+ - The database schema has been changed (#544). Create a new
```

Comment:
I'm not convinced the database schema should be considered a public API. I think it's a better idea to tout Zonemaster::Backend::DB as the API and treat the database schema as encapsulated internal state.

Reply:
The important things is to lift it up.